fern/definition/domaintakeover.yml

```diff
@@ -0,0 +1,30 @@
+types:
+  Fingerprint:
+    properties:
+      cicdPass: boolean
+      cname: list<string>
```

Comment:
Does this need to be optional?

Reply:
No, this struct isnt being sent back in the report but defines the struct for the `fingerprints.json` file. Sam suggested I fernify it

Reply:
Oh perfect. I love that. Just making sure given the other bugs we'd hit when we serialize on the wire.